examples/systemcalls/systemcalls.c

```diff
@@ -16,6 +20,12 @@ bool do_system(const char *cmd)
  *   and return a boolean true if the system() call completed with success 
  *   or false() if it returned a failure
 */
+    int ret;
+
+    ret = system(cmd);
+
+    if(ret == -1)
```

Comment:
@vishalraj3112  think about what would happen if the cmd returned with an error code other than -1 here.